force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -291,7 +331,7 @@ public with sharing class VariablesDataTypesOperators {
      * @return A Decimal representing the monthly paycheck.
      */
     public static Decimal calculateMonthlyPaycheck(Decimal hourlyRate, Double numberOfHours) {
-        return null; // Replace null with the variable you used to store the result
+        return (hourlyRate*numberOfHours) * 4; // Replace null with the variable you used to store the result
```

Comment:
Try to create a variable for the number 4, you can reuse the variable later and only need to change it once.